cli/cmd/doc.go

```diff
@@ -244,6 +244,14 @@ func generateAnnotationsDocs() []annotationDoc {
 			Name:        k8s.ProxyInboundConnectTimeout,
 			Description: "Inbound TCP connection timeout in the proxy",
 		},
+		{
+			Name:        k8s.ProxyOutboundDiscoveryCacheTimeout,
+			Description: "Maximum time allowed before an unused outbound discovery result is evicted from the cache",
+		},
+		{
+			Name:        k8s.ProxyInboundDiscoveryCacheTimeout,
+			Description: "Maximum time allowed before an unused inbound discovery result is evicted from the cache",
```

Comment:
nit: should these descriptions include the default value?

Reply:
Good call, add them in.